SWLOR.Game.Server/Service/CraftService/RecipeType.cs

```diff
@@ -2387,9 +2387,9 @@ public enum RecipeType
         InstructionShielding4 = 4686,
         InstructionInfusion1 = 4687,
         InstructionInfusion2 = 4688,
-        Missile3 = 4689,
-        Missile10 = 4690,
-        Missile25 = 4691,
+        Missile5 = 4689,
```

Comment:
You can't rename them once we've released. It'll cause an error for anyone who has already crafted these. You need to set up a migration script for it.

Reply:
I mean it would be easier to just leave the old names, they just are backend pointing at them i just figured you would want them to  match the quantities for the craft.

Reply:
Alternatively keep the existing recipe and add these new options, adjusting levels & materials appropriately.

Reply:
> I mean it would be easier to just leave the old names, they just are backend pointing at them i just figured you would want them to match the quantities for the craft.

Yea I do want them to match but you can't make that change without breaking existing character files. So either a migration script or introduce them as new recipes

Reply:
Yeah i'll just revert the originals in the recipe type and the craft and just add the new ones. Less headaches!